crates/sui-core/src/authority_server.rs

```diff
@@ -147,6 +154,7 @@ impl AuthorityServer {
 pub struct ValidatorService {
     state: Arc<AuthorityState>,
     consensus_adapter: ConsensusAdapter,
+    _checkpoint_consensus_handle: Option<JoinHandle<()>>,
```

Comment:
Hm, this is not going to be so simple, since we will need to restart it upon reconfiguration? We will work out the right structure down the line.